datafusion/sqllogictest/test_files/string/string_view.slt

```diff
@@ -396,8 +396,9 @@ EXPLAIN SELECT
 FROM test;
 ----
 logical_plan
-01)Projection: test.column1_utf8view LIKE Utf8View("foo") AS like, test.column1_utf8view ILIKE Utf8View("foo") AS ilike
-02)--TableScan: test projection=[column1_utf8view]
+01)Projection: __common_expr_1 AS like, __common_expr_1 AS ilike
```

Comment:
Nice!

Though I think this test's meaning is now changed it is supposed to be verifying cast  exprs for like

Perhaps you can change the patterns to `like '%foo%'`

Reply:
Agreed. I did some tests. I think they still use the native StringView implementation for pattern matching.
```
query TT
EXPLAIN SELECT
  column1_utf8view like '%foo%' as "like",
  column1_utf8view ilike '%foo%' as "ilike"
FROM test;
----
logical_plan
01)Projection: test.column1_utf8view LIKE Utf8View("%foo%") AS like, test.column1_utf8view ILIKE Utf8View("%foo%") AS ilike
02)--TableScan: test projection=[column1_utf8view]
```

Reply:
Done